src/core/mac/sub_mac.cpp

```diff
@@ -253,10 +253,11 @@ void SubMac::HandleReceiveDone(RxFrame *aFrame, otError aError)
 #if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE
```

Comment:
```suggestion
#if OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE && OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE
```

Reply:
It would be checked that, when `OPENTHREAD_CONFIG_MAC_CSL_DEBUG_ENABLE`  is enabled, `OPENTHREAD_CONFIG_MAC_CSL_RECEIVER_ENABLE` has to be enabled when compiling: https://github.com/openthread/openthread/blob/master/src/core/config/openthread-core-config-check.h#L520